src/components/input/Input.tsx

```diff
@@ -45,8 +47,10 @@ const Input: React.FC<InputProps> = ({
         </label>
       ) : null}
       {hint ? <Hint id={id ? `${id}-label` : undefined}>{hint}</Hint> : null}
-      {error && typeof error === 'string' ? (
-        <ErrorMessage id={id ? `${id}-error` : undefined}>{error}</ErrorMessage>
+      {errorMessageProps && errorMessageProps['aria-live'] !== 'off' ? (
+        <ErrorMessage id={id ? `${id}-error` : undefined} {...errorMessageProps}>
+          {error}
+        </ErrorMessage>
```

Comment:
This could lead to some weird behaviour, as `errorMessageProps` will always be truthy unless specified otherwise, meaning that the `ErrorMessage` will always be rendered to the DOM regardless of if there is an error or not.
I would feel better if the existing error criteria were kept (i.e. if `error` is truthy and `typeof error === "string"`) as this would result in more semantically correct HTML - Then the `errorMessageProps` can still be passed to the `ErrorMessage` component. I'm up for discussion on this though, if people feel differently.